### ISSUE TITLE
Load and parse V-HACD WRL file as Trimesh works fine but..

### ISSUE THREAD

**Issue by blendermen** [schteppe/cannon.js#333]
Can anyone see what the problem is: 
preview: http://szkola-test.esy.es/index.html 
github: https://github.com/blendermen/LoadWRLtoCannonJS
Objects are jumping around the scene after bouncing from the ground.

I imported model "monster.dae." to blender. Then I exported it through the [v-hacd ](https://github.com/kmammou/v-hacd) as a *.wrl model 

![2](https://cloud.githubusercontent.com/assets/10270475/23304594/e0956466-fa9a-11e6-8f14-edb745bc231c.jpg)

The exported model *.wrl consists of several smaller parts. From every part I pulled vertices and faces. Then I put everything in the function CANNON.Trimesh () and have combined them into a single object(compund). The final wireframe looks fine so it should work but objects just jumping around the scene. 
interesting, that standard box geometry (box2) bouncing strange too. What am i doing wrong?
![1](https://cloud.githubusercontent.com/assets/10270475/23304375/bc81f5d6-fa99-11e6-8329-31365fc04124.JPG)

**Comment by robotjsorg** [comment]
How did you get ".wrl" output? I get ".out".

**Comment by swift502** [comment]
Can't test anything out at the moment, but [this line](https://github.com/blendermen/LoadWRLtoCannonJS/blob/53729e08e8ead34769893f11a1a8e6facd221c21/index.html#L90) seems suspicious.

`groundBody.angularVelocity.set(0,10,0);`

You're setting velocity to a static object?

Edit: I think there's an optimization that skips some physics calculations for static bodies, so as a result, the ground's velocity never dissipates, and your objects bounce off the ground this way because even though the ground doesn't move, it's actually constantly spinning. :smile: 

Edit2: And because it's spinning on the y axis around the center of the scene, the further away the objects get, the faster the ground spins there and the crazier the impact is. I'm pretty sure that's it.

**Comment by blendermen** [comment]
@swift502 
You were right. As you said I have changed this line and it works perfectly! I have been waiting over a year to somebody helps me solve this problem. Thank you!  
I did not notice that line.
Updated preview here http://szkola-test.esy.es/index.html

@jmcmahon443 
I use the official [v-hacd](https://github.com/kmammou/v-hacd) description and I always get *.wrl file.
Take a look at my setup (Blender 2.79)
![obraz](https://user-images.githubusercontent.com/10270475/46635314-40d21000-cb54-11e8-9a1e-39590bcf72b9.png)
